x/oracle/keeper/keeper.go

```diff
@@ -74,16 +74,49 @@ func (k Keeper) GetExchangeRate(ctx sdk.Context, denom string) (sdk.Dec, error)
 		return sdk.OneDec(), nil
 	}
 
+	var symbol string
+
+	// Translate the base denom -> symbol
+	params := k.GetParams(ctx)
+	for i := range params.AcceptList {
```

Comment:
nit, you don't have to use the index, you can just do `for _, item := range params.AcceptList`